utils/src/main/java/org/nervos/ckb/address/AddressUtils.java

```diff
@@ -23,16 +26,20 @@ public AddressUtils(Network network) {
     this.network = network;
   }
 
-  public String generate(String publicKey) {
-    // Payload: type(01) | bin-idx("P2PH") | pubkey blake160
-    String payload = TYPE + strToAscii(BIN_IDX) + blake160(publicKey);
+  public String generateFromPublicKey(String publicKey) throws AddressFormatException {
+    return generate(blake160(publicKey));
+  }
+
+  public String generate(String args) throws AddressFormatException {
```

Comment:
It seems `args` is `arg`, a single argument.
`args` still confusing, `pubkeyHash` or `blake160Hash` may more meaningful.

Reply:
![image](https://user-images.githubusercontent.com/5823268/61520677-52c8d380-aa41-11e9-95dc-3a065f8f28a2.png)
Using `args` is just be same as address RFC.